code/style.css

```diff
@@ -1,4 +1,175 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/* Header */
+.header {
+  display: flex;
+  margin: 20px;
+  width: auto;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+.logo {
+  width: 100px;
+}
+
+h1 {
+  font-family: 'Permanent Marker', cursive;
+  font-size: 50px;
+  color: #B677D4;
+  margin: 15px;
+}
+
+.navigation {
+  display: flex;
+  align-items: flex-end;
+  margin: 20px;
+}
+
+/* Main content */
+.main-container {
+display: flex;
+flex-direction: column;
+}
+
+/* Big news */
+.big-news {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: auto;
+  height: auto;
+}
+
+.sushi {
+  width: 100%;
+  height: auto;
+  margin: 0;
+  border: 1px solid;
+  padding: 6px;
+  box-shadow: 2px 8px;
+
+}
+
+h2, h3 {
+  font-family: 'Permanent Marker';
+  padding: 0%;
+  margin: 2px;
+}
+
+.text-big-news {
+  width: auto;
+  height: auto;
+  margin: 0 30px;
+  font-family: 'Roboto';
+  text-align: justify;
+}
+
+/* Line */
+hr {
+  display: block;
+  border: 2px solid #81F0A3;
+  width: auto;
+  margin: 20px;
+}
+
+/* Other news */
+.other-news {
+  display: grid;
+  grid-gap: 5px;
+  grid-template-columns: 1fr;
+  height: auto;
+  width: 100%;
+}
+
+.small-news {
+  height: auto;
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  justify-self: center;
+  margin: 15px;
+  background-color: white;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.image-other {
+  width: 100%;
+  height: 200px;
+  margin: 0;
+  object-fit: contain;
+}
+
+h4 {
+  font-family: 'Roboto';
+}
+
+.text-other-news {
+  height: auto;
+  width: auto;
+  font-family: 'Roboto';
+  margin: 0;
+  text-align: center;
+}
+
+a:link, a:visited {
+  color:black;
+  font-family: 'Permanent Marker';
+  padding: 14px 25px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+}
+
+/* Media queries */
+@media (max-width: 767px) {
+  .navigation {
+    display:none;
+  }  
+}
+
+@media (min-width: 768px) {
+  .big-news {
+    flex-direction: row;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1199px) {
```

Comment:
Could these two media queries perhaps be merged (row 129 and 135)? Does there really need to be a max-width on the second one (row 135)?